tests/test_emails.py

```diff
@@ -334,6 +337,9 @@ def setUp(self):
         super().setUp()
         self.epp = self.create_epp(SampleDisposalNotificationEmail)
 
+    def test_only_one_project(self):
+        pass
+
```

Comment:
?

Reply:
Default test in test_email needs to be re-defined in child as not required for this script